Red 3.4.2 - Changelog (#4560)
Cog-Creators/Red-DiscordBot: docs/changelog_3_4_0.rst

.. 3.4.x Changelogs

Redbot 3.4.2 (2020-10-28)
=========================

| Thanks to all these amazing people that contributed to this release:
| :ghuser:`aikaterna`, :ghuser:`Drapersniper`, :ghuser:`jack1142`, :ghuser:`Kowlin`, :ghuser:`PredaaA`, :ghuser:`Stonedestroyer`

Read before updating
--------------------

1. Information for Audio users that are using an external Lavalink instance (if you don't know what that is, you should skip this point):

    Red 3.4.2 uses a new Lavalink jar that you will need to manually update from `our GitHub <https://github.com/Cog-Creators/Lavalink-Jars/releases/tag/3.3.1.4_1128>`_.

End-user changelog
------------------

- Fixed ``[p]massban`` not working for banning members that are in the server (:issue:`4556`, :issue:`4555`)

> - Fixed ``[p]local search`` command (:issue:`4553`)

Reviewer: ```suggestion
- Fixed the ``[p]local search`` command (:issue:`4553`)
```

- Fixed random "Something broke when playing the track." errors for YouTube tracks (:issue:`4559`)

> - Added info about metadata file to ``redbot --debuginfo`` (:issue:`4557`)

Reviewer: ```suggestion
- Added info about the metadata file to ``redbot --debuginfo`` (:issue:`4557`)
```

> - Added error messages for exceeded YouTube quota in Streams cog (:issue:`4552`)

Reviewer: ```suggestion
- Added error messages when exceeding the YouTube quota in the Streams cog (:issue:`4552`)
```

> - Improved logging for unexpected errors in Streams cog (:issue:`4552`)

Reviewer: ```suggestion
- Improved logging for unexpected errors in the Streams cog (:issue:`4552`)
```
